frontend/src/components/StyleSheet/StyleSheet.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
```

Comment:
Is this component going to be removed when we merge the stylesheet?

Reply:
I was thinking we could leave it for now so that it's easy to see how the classes from the stylesheet should be implemented before removing it